src/reducers/deleteThought.ts

```diff
@@ -107,7 +110,7 @@ const deleteThought = (state: State, payload: { path?: Path }) => {
           : parentOf(path).concat(next() as Child)
         ), { offset: 0 }] :
         // Case III: delete last thought in context; set cursor on context
-        thoughts.length > 1 ? [rootedParentOf(state, path), { offset: head(context).length }]
+        thoughts.length > 1 ? [rootedParentOf(state, path), { offset: (getTextContentFromHTML(contextHeadValue) || contextHeadValue).length }]
```

Comment:
I'm seeing this guard expression used a lot: `(getTextContentFromHTML(contextHeadValue) || contextHeadValue)`

However, is there ever a case where `getTextContextFromHTML` could return `null`? We are setting the `innerHTML` of the dummy div to a string, so it has to have `textContext`. Maybe you can have it return `return dummyDiv.textContent!` and avoid the guards upstream.

Reply:
Sure. That makes sense.